examples/lifecycle-hooks/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.15 as builder
+FROM golang:1.17 as builder
```

Comment:
Can we bump it to 1.8? integration/examples/lifecyle-hooks/Dockerfile points to 1.8.

Reply:
good idea, done!